includes/class-wc-stripe-webhook-handler.php

```diff
@@ -1136,12 +1136,29 @@ public function process_setup_intent( $notification ) {
 			return;
 		}
 
-		if ( ! $order->has_status(
-			apply_filters(
-				'wc_gateway_stripe_allowed_payment_processing_statuses',
-				[ OrderStatus::PENDING, OrderStatus::FAILED ]
-			)
-		) ) {
+		$allowed_payment_processing_statuses = apply_filters_deprecated(
```

Comment:
As above, it may be worth setting the default values explicity before we apply any filters.
```suggestion
		$allowed_payment_processing_statuses = [ OrderStatus::PENDING, OrderStatus::FAILED ];

		$allowed_payment_processing_statuses = apply_filters_deprecated(
```